### ISSUE TITLE
[opengl] some gl functions are declared (prototypes are there) but not defined (no body anywhere)

### ISSUE THREAD

**Issue by Dragorn421** [DragonMinded/libdragon#390]
`glCopyTexImage1D` and 2D have prototypes but aren't defined anywhere
```c
// weird code for the purpose of having the symbols referenced and not optimized out
int* ptr = (void*)glCopyTexImage1D;*ptr = 0;
int* ptr2 = (void*)glCopyTexImage2D;*ptr2 = 0;
```
errors on linking:
```
/opt/libragon/lib/gcc/mips64-elf/13.1.0/../../../../mips64-elf/bin/ld: (.text.startup.main+0x3b0): undefined reference to `glCopyTexImage1D'
/opt/libragon/lib/gcc/mips64-elf/13.1.0/../../../../mips64-elf/bin/ld: (.text.startup.main+0x3b4): undefined reference to `glCopyTexImage2D'
```

@rasky mentioned on discord these two functions may be implemented and supported on N64

This "missing definition" issue is also the case for the following functions:
```
glTexSubImage1D, glTexSubImage2D, glCopyTexSubImage1D, glCopyTexSubImage2D, glGetTexLevelParameter*
```

I mention this because usually unsupported methods look like: `#define glEdgeFlag(flag) _GL_UNSUPPORTED(glEdgeFlag)`, which fails a static assert when used

**Comment by Dragorn421** [comment]
Also: `glScaled`, `glRotated`, `glMultMatrixd`

**Comment by rasky** [comment]
Fixed by #494
